CHANGELOG.md

```diff
@@ -5,13 +5,15 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 ### Unreleased
 
 ### Added
+- new `options:` field in a YAML rule to enable/disable certain features
```

Comment:
Should we also add this to the docs?

Reply:
Yes i need to update lots of things in the docs ...